requirements-dev.txt

```diff
@@ -1,8 +1,9 @@
+PyYAML<5.1 # pre-commit 0.8.0 doesnt support PyYAML>=6.0 and docker-compose 1.7.0 requires PyYAML<4,>=3.10
```

Comment:
`PyYAML<4,>=3.10` but we ask for `PyYAML<5.1` and docker-compose isn't in this requirements file?

Reply:
Changed PyYAML version

Reply:
docker-compose is defined in tox.ini

Reply:
It'd be nice to upgrade docker-compose and not be stuck with such an old version of PyYAML, but let's move forward for now.